### PR TITLE
Update business-travel-insurance.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Grammar fixes and added URL to final line as it was missing (URL to be confirmed).

### CODE REVIEW

File: operations/workplace/people/working-at-mattermost/travel/business-travel-insurance.md [hunk 1]
```diff
@@ -19,9 +19,8 @@ While traveling, it is important you have on-hand at all times your [Employee ID
...
+If you are traveling on behalf of Mattermost and need a visa application, please complete the [Visa Travel Letter Generation Form](https://drive.google.com/file/d/1Eqo4Y7jses8p1uUPfOhxnb4Rp3RMXlCW/view?usp=sharing). It is important to note that you're responsible for knowing visa requirements for travel from your country of origin. If you require an invitation letter or other documents specific to your consulate, please route all questions and requests to `People@mattermost.com`.
 
 ## Reporting A Claim
 
-For type of coverage and to report a claim, refer to the [Guide to Reporting Claims](https://drive.google.com/file/d/16EhtGmq16dHe2JVsKjlyXbCNZgUmKW3j/view?usp=sharing). For further detail on the type of claims covered, refer to the
-
+For type of coverage and to report a claim, refer to the [Guide to Reporting Claims](https://drive.google.com/file/d/16EhtGmq16dHe2JVsKjlyXbCNZgUmKW3j/view?usp=sharing). For further detail on the type of claims covered, refer to the [AIG claim guide](https://www.aig.com/individual).
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/travel/business-travel-insurance.md:26]: I've added the AIG website as a placeholder. Need to confirm the intended link.

Comment (jasonblais) [reply operations/workplace/people/working-at-mattermost/travel/business-travel-insurance.md:26]: @justinegeffen Are you open to asking Finance team in community.mattermost.com, at-mentioning Isabel?

Comment (jasonblais) [reply operations/workplace/people/working-at-mattermost/travel/business-travel-insurance.md:26]: @justinegeffen have you received confirmation from Finance team?

Comment (justinegeffen) [reply operations/workplace/people/working-at-mattermost/travel/business-travel-insurance.md:26]: I completely missed this comment, @jasonblais, apologies for that. I have requested confirmation and will update as soon as I have it.

**Comment by justinegeffen** [comment]
@TQuock, would you be open to confirming this information?

**Comment by justinegeffen** [comment]
Gentle reminder about this PR, @TQuock. :)

**Comment by jasonblais** [comment]
@justinegeffen Wondering if we'd merge this PR with the grammar fixes and open a new issue to confirm the missing URL?

**Comment by justinegeffen** [comment]
@jasonblais, yeah I'm happy with that. :)

**Review by jasonblais — APPROVED** [review]
<redacted>

**Pull request merged**